unit_tests/structural_design/element/quad_hexahedron_test.cpp

```diff
@@ -305,31 +305,53 @@ BOOST_AUTO_TEST_SUITE( sd_quad_hexahedron_test )
 
 		Eigen::VectorXd loads;
 		loads.setZero(DOFCount);
-		loads(7)  = -10; // node 5, force in z-direction
-		loads(10) = -10; // node 6, force in z-direction
-		loads(13) = -10; // node 7, force in z-direction
-		loads(16) = -10; // node 8, force in z-direction
+		loads(7)  = -1e5; // node 5, force in z-direction
+		loads(10) = -1e5; // node 6, force in z-direction
+		loads(13) = -1e5; // node 7, force in z-direction
+		loads(16) = -1e5; // node 8, force in z-direction
```

Comment:
Is there a specific reason for changing the magnitudes here? If so, is it relevant to mentioned it in the code?

Reply:
No specific reason, just to get rid of the really small displacements actually.